.gitmodules

```diff
@@ -7,6 +7,9 @@
 [submodule "third_party/cpp-btree"]
 	path = third_party/cpp-btree
 	url = https://github.com/mrks/cpp-btree
+[submodule "third_party/compact_vector"]
+	path = third_party/compact_vector
+	url = https://github.com/gmarcais/compact_vector.git
```

Comment:
change back to my fork if they don't accept my PR

Reply:
Let's wait for a couple of days (Wednesday?) and see if they react.

Reply:
I am fine with using your fork. I'll switch later when your PR is merged in the compact_vector repository.

Reply:
Ah nice, it's been merged.

Reply:
used their new release. I can't validate that It works now because my boost is still being weird. If you have the time, try to compile with `-DCMAKE_DISABLE_PRECOMPILE_HEADERS=On`

Reply:
Tried the last commit and forgot updating the submodules. Did not work. Updated the submodules. Worked. 👍